README.md

```diff
@@ -308,7 +308,7 @@ For a list of free hacking books available for download, go [here](https://githu
  * [WeChall](http://www.wechall.net/)
  * [CTF archives (shell-storm)](http://shell-storm.org/repo/CTF/)
  * [Rookit Arsenal](https://amzn.com/144962636X) - OS RE and rootkit development
- * [Pentest Cheat Sheets](https://github.com/coreb1t/awesome-pentest-cheat-sheets) - Collection of cheat sheets useful for pentesting
```

Comment:
How many people find this useful information

Reply:
I started the original repo few years ago. It has currently 3.8K stars on github and was bookmarked from many other resources.
https://github.com/coreb1t/awesome-pentest-cheat-sheets

Just check the google search: https://www.google.com/search?hl=en&q=https%3A%2F%2Fgithub.com%2Fcoreb1t%2Fawesome-pentest-cheat-sheets

You will find many resources linked to the original awesome-pentest-cheat-sheets repo. I discontinue to maintain this repo and moved it to a new home: https://github.com/ByteSnipers/awesome-pentest-cheat-sheets

That is the same repo from the same author. I just updated the link to the most updated and maintained version. That is why I sent a PR to update the link.